packages/pipeline-editor/package.json

```diff
@@ -37,8 +37,8 @@
   },
   "dependencies": {
     "@elyra/metadata-common": "^2.3.0-dev",
-    "@elyra/pipeline-editor": "^0.7.0",
-    "@elyra/pipeline-services": "^0.7.0",
+    "@elyra/pipeline-editor": "https://pkg.csb.dev/elyra-ai/pipeline-editor/commit/42fac9a8/@elyra/pipeline-editor",
+    "@elyra/pipeline-services": "https://pkg.csb.dev/elyra-ai/pipeline-editor/commit/42fac9a8/@elyra/pipeline-services",
```

Comment:
is there a reason you're using the snapshots instead of the dev build? (just curious)

Reply:
I was using the dev build when testing locally, but I just wanted to make sure everything still worked in ci with the bundled version